events/controller/status.go

```diff
@@ -32,8 +32,7 @@ var (
 
 // ConditionStatusPublisher defines an interface for publishing status updates for conditions.
 type ConditionStatusPublisher interface {
-	Publish(ctx context.Context, serverID string, state condition.State, status json.RawMessage) error
-	UpdateTimestamp(ctx context.Context)
+	Publish(ctx context.Context, serverID string, state condition.State, status json.RawMessage, tsUpdateOnly bool) error
```

Comment:
I think I like having separate APIs here. What prompted your change?

Reply:
One less API route, client method to deal with mainly. Im open to adding it back in if it seems less clutter going head.